.babelrc

```diff
@@ -1,6 +1,12 @@
 {
-  "stage": 1,
-  "loose": ["all"],
-  "optional": ["runtime"],
-  "auxiliaryComment": "istanbul ignore next"
+  "presets": ["latest"],
+  "plugins": [
+    ["transform-runtime", {
+      "polyfill": true,
+      "regenerator": true
+    }],
+    ["babel-plugin-transform-builtin-extend", {
+      "globals": ["Error"]
```

Comment:
Breaking change between Babel 5 and 6: https://github.com/babel/babel/issues/3083

We probably shouldn't to that apparently.  
https://github.com/SassDoc/sassdoc/blob/master/src/errors.js